docs/03 - Implementing the Anchor Server/Communication/Events Schema.yml

```diff
@@ -102,92 +103,46 @@ components:
+            pending_anchor,
+            pending_trust,
+            pending_user,
+            no_market,
+            too_small,
+            too_large,
+            # SEP31
+            pending_sender,
+            pending_receiver,
+            pending_transaction_info_update,
+            pending_customer_info_update
           ]
-        status_change:
-          $ref: '#/components/schemas/StatusChange'
-        amount_expected:
-          $ref: '#/components/schemas/Amount'
-        amount_in:
-          $ref: '#/components/schemas/Amount'
-        amount_out:
-          $ref: '#/components/schemas/Amount'
-        amount_fee:
-          $ref: '#/components/schemas/Amount'
-        quote_id:
-          type: string
-          description: "The id of the [SEP-38](https://stellar.org/protocol/sep-38) firm quote used for the transaction."
-        started_at:
-          type: string
-          format: date-time
-        updated_at:
-          type: string
-          format: date-time
-        completed_at:
-          type: string
-          format: date-time
-          description: "When the transaction was marked as `completed` or when it was fully refunded."
-        transfer_received_at:
-          type: string
-          format: date-time
-          description: "When the Stellar transaction related to this object was received."
         message:
           type: string
           description: "This field may contain any clarifying message related to this TransactionEvent or to the transaction itself."
-        refunds:
-          $ref: '#/components/schemas/Refunds'
-        stellar_transactions:
-          type: array
-          items:
-            $ref: '#/components/schemas/StellarTransaction'
-        external_transaction_id:
-          type: string
-          description: "The id of an external transaction to be handled by the Anchor Backend."
-        custodial_transaction_id:
-          type: string
-        source_account:
-          type: string
-          description: "The Stellar account whose balance was debited in the payment that originated this event. Should be present if this event was originated from a Stellar payment."
-        destination_account:
-          type: string
-          description: "The Stellar account who received the payment that originated this event. Should be present if this event was originated from a Stellar payment."
-        customers:
-          type: object
-          description: |
-            The Identification info of the sending and receiving customers. If they were created through [SEP-12](https://stellar.org/protocol/sep-12),
-            this object should contain the SEP-12 customer `id`. Otherwise, the `account` address of the customer.
-          properties:
-            sender:
-              $ref: '#/components/schemas/StellarId'
-            receiver:
-              $ref: '#/components/schemas/StellarId'
-        creator:
-          $ref: '#/components/schemas/StellarId'
```

Comment:
Are you sure you want to remove any references from the customers related to this event? This information cannot be retrieved any other way in the future by the Anchor, can it?

Reply:
We will add the `transaction` as the payload of the transaction event as you have suggested. 

In that case, all these fields will be available in the transaction event schema.